docs/cypress/e2e/content.cy.js

```diff
@@ -11,7 +11,11 @@ describe('Cypress Test of content page', () => {
   it('check for all the links it redirects', () => {
     navLink.forEach((page) => {
       cy.visit(page);
-      cy.linkVisit();
+      cy.get('[data-test=Docs-inner--container]').then((container) => {
```

Comment:
Can we please move this test into a common function in the `commands.ts` file? This similar test is used in many places.

Reply:
done